Dockerfile

```diff
@@ -53,37 +57,8 @@ RUN usermod -d /home www-data \
     && rm -rf /var/lib/apt/lists/* \
     && pip install -U pip \
     && pip install setuptools==37.0.0 \
-    && mkdir -p /code
-
-ENV LIBREOFFICE_VERSION 6.1.6
```

Comment:
One note: the diff is misleading. I think the change here is removing installing Libreoffice from Libreoffice here but from Debian using `apt-get` many lines above.